config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-end
+    get 'welcome/index'
+
+    root 'welcome#index'
```

Comment:
This is not the way we write root controller, please have another look at the routing demo

Reply:
We have re-written it now